src/components/forms/forms.js

```diff
@@ -232,6 +255,35 @@ const formInputSelect = (config) => {
   ]);
 };
 
+const formInputRadioButtons = (config) => {
+  const {
+    id, disabled, error,
+    selectOptions, formValue, ariaDescribedby,
+  } = config;
+
+  return div({}, [
+    selectOptions.map((option) =>
+      RadioButton({
+        'aria-describedby': ariaDescribedby,
+        id: `radio_option_${id}_${option.label}`,
+        defaultChecked: formValue === option.value,
+        onClick: () => onFormInputChange(config, option.value),
+        disabled,
+        description: option.label,
+        checkedStyle: {
+          backgroundColor: '#0948B7',
+          border: '5px solid white',
+          boxShadow: '#0948b7 0 0 0 1px'
+        }
+      })
+    ),
+    error && div({ className: `error-message fadein`}, [
```

Comment:
Since this repeats multiple times through the code, can we pull this out into a separate function? Sorry, that was my bad!